pyproject.toml

```diff
@@ -141,7 +141,7 @@ exclude = [
 ignore = [
   "icepyx/quest/*",
   "icepyx/core/auth.py",
-  "icepyx/core/is2ref.py",
+  # "icepyx/core/is2ref.py",
```

Comment:
```suggestion

```
Looks like we've just removed the other ignores rather than commenting them.

Reply:
Not sure why I may have done that :laughing: :shrug: